conda/libmesh/build.sh

```diff
@@ -67,7 +67,8 @@ EOF`
                      --with-vtk-lib=${BUILD_PREFIX}/libmesh-vtk/lib \
                      --with-vtk-include=${BUILD_PREFIX}/libmesh-vtk/include/vtk-${SHORT_VTK_NAME} \
                      --with-methods="opt oprof devel dbg" \
-                     --without-gdb-command
+                     --without-gdb-command \
+                     --with-cxx-std=2014
```

Comment:
Need to change doco, such as https://mooseframework.inl.gov/getting_started/installation/hpc_install_moose.html that mentions C++11

Reply:
Thanks @WilkAndy! Good catch!